scripts/beautify_diagnostics.py

```diff
@@ -1,11 +1,13 @@
 """Script to beautify diagnostics output."""
 import json
+from os.path import abspath, dirname
 import sys
 
 from chip.clusters.ClusterObjects import ALL_ATTRIBUTES, ALL_CLUSTERS
 import yaml
 
-from matter_server.client.models.device_types import ALL_TYPES
+sys.path.insert(1, dirname(dirname(abspath(__file__))))
```

Comment:
Why is this needed?

Reply:
so you can run the script without the lib installed, just from your project root

Reply:
Meh, I don't want to on my dev machine

Reply:
Install with:

```
pip install -e .[server]
```

That will symlink the folder.

Reply:
Unfortunately that does not work on MacOS. It actually works when sqitching from zsh shell to bash but then the next issue is the wheels requirement which can not be met so for now we'll have to leave the workaround in place